pandas/tests/sparse/test_series.py

```diff
@@ -1379,11 +1379,19 @@ def test_numpy_func_call(self):
         # numpy passes in 'axis=None' or `axis=-1'
         funcs = ['sum', 'cumsum', 'var', 'mean',
                  'prod', 'cumprod', 'std', 'argsort',
-                 'argmin', 'argmax', 'min', 'max']
+                 'min', 'max']
         for func in funcs:
             for series in ('bseries', 'zbseries'):
                 getattr(np, func)(getattr(self, series))
 
+    def test_deprecated_numpy_func_call(self):
+        funcs = ['argmin', 'argmax']
```

Comment:
Can you add a comment that we need to include them again in the test above when behaviour of argmin/argmax is changes? (to make sure we don't just delete the test when the deprecation is removed)